pos_printer_network/static/description/index.html

```diff
@@ -81,34 +99,49 @@ <h3 class="oe_slogan">How It Works</h3>
       <p class="oe_mt32">
 		<ul>
 		  <font size="3px">
-          <li> Specify "Printer Name" </li>
-		  <li> Specify IP address of network printer </li>
-		  <li> Check the "Network Printer" box </li>
-		  <li> Select product categories and click on "Save" </li>
+              <li> Specify <b>Printer Name</b> </li>
+              <li> Specify <b>IP address</b> of network printer </li>
+              <li> Check the <b>Network Printer</b> box </li>
+              <li> Select <b>Printed Product Categories</b> and click on "Save" </li>
 		  </font>
 		</ul>
 	  </p>
     </div>
 	<div class="oe_row_img oe_centered">
       <img class="oe_demo oe_picture oe_screenshot" src="order_printer_form.png?1"/>
     </div>
+          <div class="oe_span12">
+      <p class="oe_mt32">
+		<ul>
+		  <font size="3px">
+              <li> Go to <em>Point of Sale &rarr; Configuration &rarr; Point of Sale</em> and open POS config form.</li>
+              <li> Add an item (network printer) in the <b>Order Printers</b> menu and click on "Save"</li>
+		  </font>
+		</ul>
+	  </p>
+    </div>
+      	<div class="oe_row_img oe_centered">
+        <img class="oe_demo oe_picture oe_screenshot" src="pos_config2.png"/>
+    </div>
   </div>
 </section>
 
 <section class="oe_container oe_dark">
   <div class="oe_row oe_spaced">
+      <h4 class="oe_slogan">Receipt Printer</h4>
     <div class="oe_span12">
       <p class="oe_mt32 text-center">
-        <font size="3px">Go to <em>Point of Sale &rarr; Configuration &rarr; Point of Sale</em> and open POS config form.
+        <font size="3px">
 		<ul>
+        <li> Go to <em>Point of Sale &rarr; Configuration &rarr; Point of Sale</em> and open POS config form.</li>
 		  <li>
-			Check the "Network Printer" box at <em>Hardware Proxy/PosBox</em> section
-		  </li>
-		  <li>
-			Optional: Specify "Network Receipt Printer IP" to be able to print POS receipts
+              Check the <b>Network Printer</b> box at <em>Hardware Proxy/PosBox</em> section
 		  </li>
+        <li>
+            Switch <b>Printer Type</b> to "Network Printer"
```

Comment:
It will be better to mention that new field ``Printer Type`` has 2 values. 
For example:
 **Printer Type** has 2 values: ``USB Printer`` and ``Network Printer``. Switch it to ``Network Printer``